ansible_ai_connect/main/tests/test_views.py

```diff
@@ -173,3 +173,18 @@ def test_protected_access_aap_system_auditor(self):
         self.client.force_authenticate(user=self.user)
         r = self.client.get(reverse("prometheus-metrics"))
         self.assertEqual(r.status_code, HTTPStatus.OK)
+
+@modify_settings()
+@override_settings(WCA_SECRET_BACKEND_TYPE="dummy")
+class TestMarkdownMe(TestCase):
+    @override_settings(ANSIBLE_AI_ENABLE_TECH_PREVIEW=False)
```

Comment:
You can remove the `@override_settings(ANSIBLE_AI_ENABLE_TECH_PREVIEW=False)` line. We don't need it.